files/ru/learn/server-side/express_nodejs/mongoose/index.html

```diff
@@ -50,7 +50,7 @@ <h3 id="Каков_наилучший_способ_взаимодействия_
 <p>Преимущество применения ORM состоит в том, что программисты могут сосредоточиться на объектах JavaScript, а не  на семантике базы данных —  особенно, если требуется работать с разными базами данных (на одном или разных веб-сайтах). Они также дают очевидное место для валидации и проверки данных.</p>
 
 <div class="note">
-<p>Совет:  Применение ODM / ORMs часто приводит к снижению затрат на разработку и обслуживание! Если вы не очень хорошо знакомы с родным языком запросов или если производительность имеет первостепенное значение, следует серьёзно рассмотреть возможность применения ODM.</p>
+<p>Совет:  Применение ODM / ORMs часто приводит к снижению затрат на разработку и обслуживание! Если вы не очень хорошо знакомы с родным языком запросов или если производительность не имеет первостепенного значения, следует серьёзно рассмотреть возможность применения ODM.</p>
```

Comment:
"Родной язык запросов" лучше заменить так:

```suggestion
<p>Совет:  Применение ODM / ORMs часто приводит к снижению затрат на разработку и обслуживание! Если вы не очень хорошо знакомы с языком запросов базы данных или если производительность не имеет первостепенного значения, следует серьёзно рассмотреть возможность применения ODM.</p>
```